chapters/extend/executor.rst

```diff
@@ -105,14 +141,12 @@ Remember to add ``super().__init__(*args, **kwargs)`` to your :meth:`__init__`.
     All attributes declared in :meth:`__init__` will be persisted during :meth:`save`  and :meth:`load`.
 
 
-Override :meth:`post_init`
-^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 So what if the data you need to load is not in simple type. For example, a deep learning graph, a big pretrained model, a gRPC stub, a tensorflow session, a thread? The you can put them into :meth:`post_init`.
 
-Another scenario is when you know there is a better persistence method other than ``pickle``. For example, your hyperparameters matrix in numpy ``ndarray`` is certainly pickable. However, one can simply read and write it via standard file IO, and it is likely more efficient than ``pickle``. In this case, you do the data loading in :meth:`post_init`.
+Another scenario is when you know there is a better persistence method other than ``pickle``. For example, your hyperparameters matrix in numpy ``ndarray`` is certainly pickable. However, you can simply read and write it via standard file IO, and it is likely more efficient than ``pickle``. In this case, you do the data loading in :meth:`post_init`.
```

Comment:
I would emphasize you are still talking about post_init.
Something like
```When there is a better persistence method other than ``pickle`` is also interesting defining a custom  :meth:`post_init`.
```
instead of  
```
Another scenario is when you know there is a better persistence method other than ``pickle``.
```

Reply:
changed to `It is also interesting to override :meth:`post_init` when there is a better persistence method other than pickle.`